recipes/libucl/all/patches/0002-0.8.1-cmake-add-install+use-find_package.patch

```diff
@@ -0,0 +1,147 @@
+- Fixes dll/static cmake builds
+- Fixes libcurl
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -4,13 +4,13 @@
+ SET(LIBUCL_VERSION_MAJOR 0)
+ SET(LIBUCL_VERSION_MINOR 5)
+ SET(LIBUCL_VERSION_PATCH 0)
+-
+-SET(LIBUCL_VERSION
+-        "${LIBUCL_VERSION_MAJOR}.${LIBUCL_VERSION_MINOR}.${LIBUCL_VERSION_PATCH}")
++SET(LIBUCL_VERSION "${LIBUCL_VERSION_MAJOR}.${LIBUCL_VERSION_MINOR}.${LIBUCL_VERSION_PATCH}")
+-
```

Comment:
This seems a little extra 🤔 why did you decide to include this?

Reply:
I did this to let this chunk  be line neutral: number of input lines == number of output lines.
That makes later chunks much easier.